### PR TITLE
User story 10 accessory slots

### PR DESCRIPTION
**Pull request by linuskar**
# Accessory slots
- Added accessory slots in inventory. 
- Accessories get saved.
- With right click, you can destroy the inventory item.
- To equip an accessory drag a valid accessory from inventory to an accessory slot.
- There is notification/warning popups for better UI feedback (timer based).

image

image

image

image

### CODE REVIEW

File: DAT257AgileProject/Assets/Scripts/UI/WarningPopup.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+
+public class WarningPopup : MonoBehaviour
+{
+    public float displayTime = 2.5f; // The time the warning will be displayed
```
Comment (Caramajau) [inline DAT257AgileProject/Assets/Scripts/UI/WarningPopup.cs:8]: Can it be private?

Comment (linuskar) [reply DAT257AgileProject/Assets/Scripts/UI/WarningPopup.cs:8]: yes

**Review by HermanForsberg — APPROVED** [review]
looks good

**Review by Caramajau — APPROVED** [review]

**Review by Figer-op — APPROVED** [review]
hahah före davoid

**Pull request merged**